prqlc/bindings/prqlc-c/examples/minimal-zig/Taskfile.yml

```diff
@@ -0,0 +1,39 @@
+# yaml-language-server: $schema=https://json.schemastore.org/taskfile.json
+
+version: 3
+
+vars:
+  project_root: "../../../../.."
+
+tasks:
+  run:
+    cmds:
+      - task: build
+      - ./zig-out/bin/minimal-zig
+
+  build-prql:
+    desc: "Build prqlc-c"
+    cmds:
+      - cargo build --package prqlc-c --release
+      - mkdir -p c/
+      - cp {{.project_root}}/prqlc/bindings/prqlc-c/prqlc.h c/
+      - cp {{.project_root}}/target/release/libprqlc_c.* c/
+
+  build:
+    desc: "Build the project"
+    cmds:
+      - zig build
```

Comment:
Is `source` and `generated` unnecessary?

Reply:
It's not necessary, though it can help with caching